packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/Advanced/SecondPasswordWallet/Settings/template.js

```diff
@@ -18,8 +20,11 @@ const ButtonWrapper = styled(ButtonGroup)`
   }
 `
 
+const validateSecondPassword = (value, allValues, props) => {
```

Comment:
This should probably go in services/FormHelper/index.js

Reply:
I think that file is for validators that are common, others are scattered closer to where they are used.